src/main/java/org/jabref/model/entry/AuthorList.java

```diff
@@ -653,6 +570,10 @@ public String getAsUnformattedLatexFree() {
             return authorsUnformattedLatexFree;
         }
 
+        // The AuthorList was not created with the factory method
+        if (authorsUnformatted == null) {
+            authorsUnformatted = getAsFirstLastNamesWithAnd();
+        }
```

Comment:
If the `AuthorList` is not created by the factory method `parse`, it was not created from a `String` and `authorsUnformatted` does not exist. An alternative would be to just throw an exception, currently that would just force the test cases to change.

There is a public constructor for `AuthorList` that is removable. The two protected constructors should also be removable if `AuthorListParser` is made an inner class but I'd consider that detrimental to readability.